pkg/logging/logger.go

```diff
@@ -46,17 +48,23 @@ const (
 )
 
 func init() {
+	mutex.Lock()
 	defaultLogConsumer = NewFilteredLevelLogConsumer(LogLevelInfo, os.Stdout)
+	mutex.Unlock()
```

Comment:
I know this was already there, but is this `init()` needed or is it preferable to set during definition?
```go
var defaultLogConsumer OptimizelyLogConsumer = NewFilteredLevelLogConsumer(LogLevelInfo, os.Stdout)
```

Reply:
since there is a SetLogger.  I actually think this is safer.